<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchReplaceTool</class>
 <widget class="QDialog" name="BatchReplaceTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BatchReplaceTool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblFindText">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Find Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="iptFindName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblReplaceText">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Replace With</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="iptReplaceName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblObjectType">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Object Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxObjectType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnDoReplace">
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
